src/Neo/SmartContract/Native/LedgerContract.cs

```diff
@@ -310,7 +310,7 @@ public Transaction GetTransaction(DataCache snapshot, UInt256 hash)
             return GetTransactionState(snapshot, hash)?.Transaction;
         }
 
-        [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates, Name = "getTransaction")]
+        [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates, Name = nameof(GetTransaction))]
         private Transaction GetTransactionForContract(ApplicationEngine engine, UInt256 hash)
```

Comment:
I won't do this. `GetTransaction` is a separate Ledger method that is not related to `GetTransactionForContract`; it's just about the fact that these methods have the same name.

Reply:
Nothing changed here, the first letter is lowecase during abi construction

Reply:
> Nothing changed here

Yes, I see, but this `nameof`-related change may lead to unwanted bugs in future. Consider if in future we rename `GetTransaction`, then Ledger's ABI will be changed which is unexpected. That's why I'd prefer to use raw string here.

Reply:
OK, let's keep it.

Reply:
We can revert it, for me it's the same